middleware/logger/default_logger.go

```diff
@@ -90,6 +90,11 @@ func defaultLoggerInstance(c fiber.Ctx, data *Data, cfg Config) error {
 			buf.WriteString("\n")
 		}
 
+		// Check if Filter is defined and call it
+		if cfg.Filter != nil && !cfg.Filter(c) {
```

Comment:
@JIeJaitt This can be moved to the beginning of the function. There's no point is running all that only to have the filter return True and skip 😂